.vscode/tasks.json

```diff
@@ -24,7 +24,7 @@
     {
       "label": "Run current file integration test",
       "type": "shell",
-      "command": "go run cmd/integration_test/main.go cli ${relativeFile}",
+      "command": "go generate pkg/integration/tests/tests.go && go run cmd/integration_test/main.go cli ${relativeFile}",
```

Comment:
this slightly increases the latency of running a test from vscode but imo it's worth it to avoid having to manually add a test to the tests list